examples/cov2d_simulation.py

```diff
@@ -168,4 +173,22 @@
 logger.info(f'Deviation of the noisy covariance estimate: {diff_covar}')
 logger.info(f'Estimated images normalized RMSE: {nrmse_ims}')
 
-
+# plot the first images at different stages
+idm = 0
+plt.subplot(2, 2, 1)
+plt.imshow(np.real(-imgs_noise[..., idm]), cmap='gray')
```

Comment:
Why minus? Why `np.real`? Shouldn't the images already be real?

Reply:
You are right. Removed in the new commit 88cd518.